### PR TITLE
[onert]Adding shape inference for binary arithmetic operation

### PR DESCRIPTION
**Pull request by YiHyunJin**
This adds shape inference for binary arithmetic operation(div, mul).
If https://github.com/Samsung/ONE/pull/1419 is merged, I will change to handleBinaryArithmeticOp.

Signed-off-by: YiHyunjin hj0412.yi@samsung.com

### CODE REVIEW

**Comment by glistening** [comment]
@hyunsik-yoon #1419 is merged and this PR is updated. PTAL.

File: tests/nnapi/specs/V1_2/div_dynamic_nnfw.mod.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# model
```
Comment (hyunsik-yoon) [inline tests/nnapi/specs/V1_2/div_dynamic_nnfw.mod.py:1]: could you add license header?

File: tests/nnapi/specs/V1_2/mul_dynamic_nnfw.mod.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# model
```
Comment (hyunsik-yoon) [inline tests/nnapi/specs/V1_2/mul_dynamic_nnfw.mod.py:1]: ditto

**Comment by hseok-oh** [comment]
@YiHyunJin Please resolve conflict

**Pull request merged**